### PR TITLE
VDS-461: Rework navigation process on checkout

### PR DESCRIPTION
**Pull request by ilyawzrd**
Video in comments: https://virtocommerce.atlassian.net/browse/VDS-461

### CODE REVIEW

**Review by trueboroda — APPROVED** [review]

File: assets/scss/custom/_navs.scss [hunk 1]
```diff
@@ -88,4 +88,16 @@
...
+.checkout-wizard-active-step {
+  > a {
+    color: $brand-primary !important;
+    border-radius: $panel-border-radius !important;
+    background-color: $body-bg !important;
+    border: 1px solid $brand-primary !important;
+    &:hover {
+      background-color: $body-bg !important;
+    }
+  }
```
Comment (asvishnyakov) [inline assets/scss/custom/_navs.scss:102]: May we instead replace it with `.nav-outline`, which will set `color`, `border` & `background` for `li.active > a` and introduce new variable `$nav-outline-border-radius`? Then this extension will come to bootstrap 4 unchanged.

File: templates/cart.liquid [hunk 2]
```diff
@@ -18,10 +18,10 @@
...
             <div class="page-header clearfix">
                 <ol class="nav nav-pills pull-left">
-                    <li role="presentation" ng-repeat="step in checkout.wizard.steps" ng-class="{'active': step.isActive}">
-                        <a href="" ng-click="checkout.wizard.goToStep(step)">
-                            <span class="h1 text-middle" ng-bind="$index + 1"></span>
-                            <u class="h2 text-middle" ng-bind="step.title"></u>
+                    <li role="presentation" ng-repeat="step in checkout.wizard.steps" ng-class="{'checkout-wizard-active-step': step.isActive, 'disabled': !step.isActive}">
+                        <a href="">
+                            <span class="h2 text-middle" ng-bind="$index + 1"></span>
+                            <u class="h2 text-decoration-none text-middle" ng-class="{'text-default-color': !step.isActive}" ng-bind="step.title"></u>
```
Comment (asvishnyakov) [inline templates/cart.liquid:24]: Use `span` instead and you will not need text-decoration-none

File: templates/cart.liquid [hunk 3]
```diff
@@ -18,10 +18,10 @@
         <div class="hidden-xs hidden-sm col-md-12 hidden-print">
             <div class="page-header clearfix">
                 <ol class="nav nav-pills pull-left">
-                    <li role="presentation" ng-repeat="step in checkout.wizard.steps" ng-class="{'active': step.isActive}">
-                        <a href="" ng-click="checkout.wizard.goToStep(step)">
-                            <span class="h1 text-middle" ng-bind="$index + 1"></span>
-                            <u class="h2 text-middle" ng-bind="step.title"></u>
+                    <li role="presentation" ng-repeat="step in checkout.wizard.steps" ng-class="{'checkout-wizard-active-step': step.isActive, 'disabled': !step.isActive}">
+                        <a href="">
```
Comment (asvishnyakov) [inline templates/cart.liquid:22]: I would like to add `py-3 mr-4` to follow design

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

File: templates/cart.liquid (hunk 3 above)
Comment (ilyawzrd) [reply templates/cart.liquid]: done

File: templates/cart.liquid (hunk 2 above)
Comment (ilyawzrd) [reply templates/cart.liquid]: done

File: assets/scss/custom/_navs.scss (hunk 1 above)
Comment (ilyawzrd) [reply assets/scss/custom/_navs.scss]: done

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**